.travis.yml

```diff
@@ -22,6 +22,8 @@ branches:
         - /^analysis-.*$/
 
 matrix:
+    allow_failures:
+        - php: hhvm
```

Comment:
we could try to detect hhvm and trigger a skip just for the tests that don't work with hhvm rather than ignore hhvm issues over the whole package.